capabilities/writetarget/beholder/info.go

```diff
@@ -11,6 +11,14 @@ type MetricInfo struct {
 	Description string
 }
 
+// ChainInfo contains the chain information (used as execution context)
+type ChainInfo struct {
+	ChainFamilyName string
+	ChainID         string
+	NetworkName     string
+	NetworkNameFull string
```

Comment:
Do we need to say `ChainInfo.ChainX`? Internally, we have used `RelayID { Network; ChainID }`, and chain selectors uses `Family`

Reply:
Maybe `ChainID` is fine since that is so familiar, but `FamilyName` could stand alone :shrug: 
```suggestion
// ChainInfo contains the chain information (used as execution context)
type ChainInfo struct {
	FamilyName string
	ChainID         string
	NetworkName     string
	NetworkNameFull string
```